airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/cdc/MySqlDebeziumOperations.kt

```diff
@@ -377,10 +377,30 @@ class MySqlDebeziumOperations(
             )
             .with("numeric.type", MySQLNumericConverter::class.java.getName())
             .with("boolean.type", MySQLBooleanConverter::class.java.getName())
+
+            val serverTimezone: String? = (configuration.incrementalConfiguration as CdcIncrementalConfiguration).serverTimezone
+            if (!serverTimezone.isNullOrBlank()) {
+                dbzPropertiesBuilder.with("database.connectionTimezone", serverTimezone)
```

Comment:
Shall we add a log to log the server timezone? 

What would happen if the serverTimezone is not provided? does that need a warning?

Reply:
I just followed current logic here: https://github.com/airbytehq/airbyte/blob/6f0fd7a2b9575a5fb4d5b0ebcfc8bc192869fdb1/airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/cdc/MySqlCdcProperties.java#L73-L86

Shouldn't be a problem because we can already run CDC without this parameter.

Reply:
sounds good